plugins/inputs/t128_lte/t128_lte.go

```diff
@@ -0,0 +1,167 @@
+package t128_lte
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const (
+	T128_CONFIG_FILE = "/var/run/128technology/config.json"
```

Comment:
is this the only input that uses this path? @gregschrock does it make sense to have some common lib code in telegraf to contain some of these generic paths. Alos for gloabl and local init paths